### ISSUE TITLE
Poor prediction results and no pretrained model

### ISSUE THREAD

**Issue by Marina-Liu** [afiosman/MR-to-CT-image-translation-using-unet#1]
Thanks for your excellent work! I am trying to reproduce the result. And I found that there are a pretrained model loading process. The upload repository didn't contain it. So I annotated these code and run the model. And I'm using the same BRATS2021 dataset. But the results is very poor, and different from the papaer result. The prediction result shows as  below, The prediction is pretty white and differs greatly from the true value.
<img width="561" alt="4434483240e9f1c11c5288a51df34c8" src="https://github.com/afiosman/dense-dilated-residual-convolutions-for-contrast-enhanced-MRI-synthesis/assets/96519520/aef51bf1-d61c-4f3f-8021-515dcf2507f8">

The training process and loss curve are list below:

<img width="507" alt="9b3fe43d14d6ab2d72a3a2a292380da" src="https://github.com/afiosman/dense-dilated-residual-convolutions-for-contrast-enhanced-MRI-synthesis/assets/96519520/89ed0555-eea8-4344-a2f2-3ef757a06c88">

<img width="245" alt="9c055a8c9b6dc06d03e315733dfecac" src="https://github.com/afiosman/dense-dilated-residual-convolutions-for-contrast-enhanced-MRI-synthesis/assets/96519520/8cd0eb90-db68-400c-8c19-0c640aeba89b">

Why this happen?  How could I reproduce the paper result performance? It will be very grateful that if you could help me solve this problem.
